packages/stripe_web/lib/src/web_stripe.dart

```diff
@@ -175,6 +181,51 @@ class WebStripe extends StripePlatform {
     return response.paymentIntent!.parse();
   }
 
+  Future<PaymentIntent> confirmIdealPayment(
+      String paymentIntentClientSecret, PaymentMethodDataIdeal paymentData,
+      {String? returnUrl}) async {
+    if (paymentData.bankName == null) throw 'bankName is required for web';
```

Comment:
I think it is better to call bankName is required for idea. since this is not specific to web